### PR TITLE
[config] clean defaults

### PR DESCRIPTION
**Pull request by degemer**
Defaults are now commented.

### CODE REVIEW

**Comment by hkaj** [comment]
Behold the sneakiness of those spaces you added: https://github.com/DataDog/docker-dd-agent/pull/100

**Comment by degemer** [comment]
Fix https://github.com/DataDog/dd-agent/issues/2558

File: datadog.conf.example [hunk 1]
```diff
@@ -36,8 +36,7 @@ api_key:
 # create_dd_check_tags: no
 
 # Collect AWS EC2 custom tags as agent tags (requires an IAM role associated with the instance)
-# collect_ec2_tags: no
-
+# collect_ec2_tags: yes
```
Comment (remh) [inline datadog.conf.example]: Default is no here

File: datadog.conf.example [hunk 2]
```diff
@@ -48,24 +47,23 @@ api_key:
 # collector_profile_interval: 20
 
 # use unique hostname for GCE hosts, see http://dtdg.co/1eAynZk
-gce_updated_hostname: yes
+# gce_updated_hostname: yes
```
Comment (remh) [inline datadog.conf.example]: Unfortunately you can't do that. As we want to keep the old defaults for existing installations (for compatibility purposes) but we want new installations to have this flag enabled

**Comment by degemer** [comment]
Thanks for the review, updated as per your comments @remh.

**Comment by gmmeyer** [comment]
This looks good to me, it's a lot better than before! 👍

**Pull request merged**